services/core-web/src/App.tsx

```diff
@@ -45,4 +44,4 @@ const App = () => {
   );
 };
 
-export default hot(module)(App);
```

Comment:
Was this what was ironically breaking the hot reloading?

Reply:
Indeeed 😅

Reply:
I think it might have broken when we removed `babel` from Webpack last year